src/empire/trade_prices.c

```diff
@@ -8,6 +8,9 @@
 #include "core/calc.h"
 #include "trade_prices.h"
 
+#define MIN_BUY_PRICE 1
+#define MIN_SELL_PRICE 1
```

Comment:
I guess this can be `MIN_PRICE` since they're both `1`.

Reply:
Fixed.